install/source/index.markdown

```diff
@@ -74,7 +74,15 @@ title: MoveIt 1 Source Build - Linux
             pip3 install git+https://github.com/catkin/catkin_tools.git
         </code>
     </div>
-    Then re-run <span class="ros-command">catkin config</span>.
+    Then re-run <span class="ros-command">catkin config</span>.</p>
```

Comment:
This doesn't look right in a couple markdown viewers I tried.  :(  Is that normal?

e.g.
https://github.com/ros-planning/moveit.ros.org/blob/7344451001b2a76e161bff15c7d497b7d268e766/install/source/index.markdown

Reply:
I guess so, master branch looks the same way

Reply:
Yea, it's html.  I don't understand how it works but it does.  You can see that if you build the website locally.